README.md

```diff
@@ -2094,7 +2099,9 @@ IEEE Trans. Pattern Anal. Machine Intell. (TPAMI), 45(2)2519:2532, 2022. [PDF](h
 - <a name="Chen22tcsvt"></a>Chen, Z., Wu, J., Hou, J., Li, L., Dong, W., Shi, G.,  
 *[ECSNet: Spatio-Temporal Feature Learning for Event Camera](https://doi.org/10.1109/TCSVT.2022.3202659)*,  
 IEEE Trans. Circuits Syst. Video Technol. (TCSVT), 2022.
-
+- <a name="Elshair23access"></a>El Shair, Z., Hassani, A., Rawashdeh, S.,   
+*[CSTR: A Compact Spatio-Temporal Representation for Event-Based Vision](https://doi.org/10.1109/ACCESS.2023.3316143)*,   
+IEEE Access, 2023. [PDF](https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=10254219), [Code](https://github.com/Zelshair/cstr-event-vision)
```

Comment:
Same comment. This 2nd link is not needed, since the official link (doi) points to the paper and the paper is open access. Both links point to the same document in IEEE.
The additional PDF link is only needed when the official PDF is behind a paywall.

Reply:
Thanks again. I have just addressed this issue in the latest commit as well.